### DISCUSSION TITLE
What does "Bind bone to node" do, and how do I use it?

### DISCUSSION THREAD

**Discussion by PhoenixStormJr** [AnimeEffectsDevs/AnimeEffects#59] [Q&A] [answered]
I can't for the life of me figure out how to use this option in AnimeEffects. What does "Bind bone to node" do? I can't figure it out! Please help!

![WhatDoesBindBoneToNodeDo](https://github.com/user-attachments/assets/236caa62-b687-45dd-9e21-0deea7f67270)

**Comment by p-yukusai** [comment, marked as answer]
Solved on the issue of the same name, closing.

**Comment by PhoenixStormJr** [comment]
I feel so stupid for not getting this sooner, but thank you! 

Note for other noobs like me:
Basically for any other noobs like me, basically put all the objects (or nodes) related to the character in a FOLDER! Basically make a folder named "Yoshi" (Or whatever you wanna name it) and put "YoshiFoot.png" (Or whatever the name is) inside the folder. Next, after all the objects (nodes) are inside the folder, select the FOLDER ITSELF, DO NOT SELECT THE NODE, and create bones on the FOLDER! Then use Bind Bone to Node. After selecting the bone, a menu will appear that normally does not appear if you simply select a node. The menu has a list of nodes to select from! Select a bone, select a menu, and walla!

> "Node" is what we call layers and folders in the code, binding a bone to a node therefore means what it says on the tin, if you move the bone you move the entire node it's bound to (i.e. the layer) . [You can see a showcase here](https://cdn.discordapp.com/attachments/630440314058047506/1147700029822349352/Lpn0N7K.gif?ex=66def3b2&is=66dda232&hm=63c767b84c6082f45c8887e27059526d69e51fd0dd9ffac2a143f137f407402a&)
